bundles/org.openhab.binding.nanoleaf/README.md

```diff
@@ -77,10 +115,34 @@ A lightpanel thing has the following channels:
+| singleTap           | Switch    | [Canvas Only] Is set when the user taps that panel once (1 second pulse)              | Yes       |
+| doubleTap           | Switch    | [Canvas Only] Is set when the user taps that panel twice (1 second pulse)              | Yes       |
+
+
+**color and panelColor**
 
 The color and panelColor channels support full color control with hue, saturation and brightness values. 
-For example, brightness of all panels can be controlled by defining a dimmer item for the color channel of the controller thing.
+For example, brightness of *all* panels at once can be controlled by defining a dimmer item for the color channel of the *controller thing*.
 The same applies to the panelColor channel of a lightpanel thing.
+
+What might not be obvious and even maybe confusing is the fact that brightness and color use the *same* channel but two different *itemTypes*. While the Color-itemtype controls the color, the Dimmer-itemtype controls the brightness on the same channel 
+
+**Touch Support**
+
+Nanoleaf's Canvas introduces a whole new experience by adding touch support to it. This allows single and double taps on individual panels to be detected and then processed via rules to further control any other device!
+Note that even gestures like up, down, left, right are sent but can only be detected on the whole set of panels and not on an individual panel. These four gestures are not yet supported by the binding but may be added in a later release.
+
+To detect single and double taps the panel's have been extended to have two additional channels named singleTap and doubleTap which act like switches that are turned on as soon as a tap type is detected.
+These switches then act as a pulse to further control anything else via rules.
+
+If a panel is tapped the switch is set to ON and automatically reset to OFF after 1 second (this may be configured in the future) to simulate a pulse. A rule can easily detect the transition from OFF to ON and later detect another tap as it is automatically reset by the binding. See the example below on Panel 2.
+
+Keep in mind that the double tap is used as an already built-in functionality by default when you buy the nanoleaf: it switches all panels (hence the controller) to on or off like a light switch for all the panels at once. To circumvent that
+
+- Within the nanoleaf app go to the dashboard and choose your device. Enter the settings for that device by clicking the cog icon in the upper right corner. 
+- Enable "Touch Gesture" and assign the gestures you want to happen but set the double tap to unassigned.
+- To still have the possibility to switch on the whole canvas device with all its panels by double tapping a specific panel, you can easily write a rule that triggers on the double tap channel of that panel and then toggles the Power Channel of the controller. See the example below on Panel 1.
+
 More details can be found in the full example below.
 
 ## Full Example
```

Comment:
The configuration in the nanoleaf.things sample file should be enhanced with the new deviceType parameter. Since the rules make use of the new channels for touch, deviceType should be to set to "canvas".

Reply:
The reason I am sure what to do with the parameter is that you had added it. What was the intention of that parameter?

Reply:
As the device type is now automatically set, the parameter has become read-only. The parameter has been added to the table in the readme.